boa/src/builtins/property/mod.rs

```diff
@@ -284,3 +278,75 @@ impl<'a> From<&'a Value> for Property {
         }
     }
 }
+
+#[derive(Trace, Finalize, Debug, Clone)]
+pub enum PropertyKey {
+    String(RcString),
+    Symbol(RcSymbol),
+}
+
+impl From<RcString> for PropertyKey {
+    fn from(string: RcString) -> PropertyKey {
+        PropertyKey::String(string)
+    }
+}
```

Comment:
```suggestion
impl From<RcString> for PropertyKey {
    #[inline]
    fn from(string: RcString) -> PropertyKey {
        PropertyKey::String(string)
    }
}
```
These are very small functions and annotating it with `#[inline]` helps the compiler out, when optimizing.